Fixes AB#1741220 - update proxy info (#6201)
MicrosoftDocs/PowerShell-Docs: reference/7.0/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

Some features may not be available on all platforms.

Because of changes in .NET Core 3.1, PowerShell 7.0 and higher use the
[HttpClient.DefaultProxy](/dotnet/api/system.net.http.httpclient.defaultproxy?view=netcore-3.1)

> Property to determine the proxy configuration.

Reviewer: ```suggestion
property to determine the proxy configuration.
```

Author: fixed

The value of this property is different rules depending on your platform:

- **For Windows**: Reads proxy configuration from environment variables or, if those are not
  defined, from the user's proxy settings.
- **For macOS**: Reads proxy configuration from environment variables or, if those are not defined,
  from the system's proxy settings.

> - **For Linux**: Reads proxy configuration from environment variables or, in case those are not

Reviewer: ```suggestion
- **For Linux**: Reads proxy configuration from environment variables or, if those are not
```

It just reads better if they are consistent to me. Feel free to ignore

Author: fixed
